python/doc/examples/calibration/bayesian_calibration/plot_gibbs_simus.py

```diff
@@ -541,30 +545,31 @@ def log_cond_tau_post(tau):
 acc_rate = (tau_post[1:] != tau_post[:-1]).mean()
 print("Acceptance rate: %s" % acc_rate)
 
-
 # %%
 # Plot posterior distribution marginals
 
 # extract interest parameters
 post_sample = sample.getMarginal([i for i in range(p + 1)])
-post_sample.setDescription(["$\\theta_1$", "$\\theta_2$", "$\\tau$"])
+post_sample.setDescription(["$\\theta_0$", "$\\theta_1$", "$\\tau$"])
 
 ks = ot.KernelSmoothing()
 posterior = ks.build(post_sample)
```

Comment:
@MerlinKeller means something like this in his comment https://github.com/openturns/openturns/pull/2081#discussion_r964744902:

```suggestion
prior_support = ot.Interval([-2.0, -2.0, 1e-4], [2.0, 2.0, 1e1])
posterior = ot.TruncatedDistribution(ks.build(post_sample), prior_support)
```
The support of the posterior distribution is necessarily included in the support of the prior (here they are the same). This is a way to correct the result of the kernel smoothing.

Reply:
oh ok